### PR TITLE
GROW-175 Styling Changes

### PR DESCRIPTION
**Pull request by eddieferrer**
* Made some styling and content changes

### CODE REVIEW

**Comment by eddieferrer** [comment]
screencapture-dev-vm-01-kiva-org-get-started-2020-09-10-15_35_03

**Comment by eddieferrer** [comment]
screencapture-dev-vm-01-kiva-org-get-started-2020-09-10-15_46_26

File: src/pages/GetStarted/GetStartedCauses.vue [hunk 1]
```diff
@@ -184,18 +182,49 @@ export default {
...
-	padding: 1.625rem 0;
-}
-
 .get-started {
+	&__form {
+		padding: 0.85rem;
+		margin: 2rem auto;
+		background-color: $white;
+		border-radius: 1rem;
+		width: 98%;
```
Comment (ryan-ludwig) [inline src/pages/GetStarted/GetStartedCauses.vue:191]: what does this do?

Comment (eddieferrer) [reply src/pages/GetStarted/GetStartedCauses.vue:191]: For mediumish screens, keeps it to being full width so the darker gray background still shows through on all sides

**Comment by ryan-ludwig** [comment]
I think this is close, just needs some tweaks on the margins to make them consistent like the zeplin file 
image

**Review by emuvente — APPROVED** [review]
👍

**Review by ryan-ludwig — APPROVED** [review]
Looks great!

**Pull request merged**